roles/services/defaults/main.yml

```diff
@@ -9,7 +9,6 @@ hydra_db: my_hydra_db
 hydra_pg_user: my_hydra_pg_user
 hydra_pg_pass: my_hydra_pg_pass
 rails_secret_key_base: eaa4e23ab500b77c8c2064780b4f41b83b0b0e7ec13036d55b72d87d8cc8d9ec9183230b7b3ccab7bc6387c73317b0fbb3c637f98b6ad265b99e7304b5e47292
-# url for 0.6.2 is fits_url: https://fits.googlecode.com/files/fits-0.6.2.zip
-fits_url: http://projects.iq.harvard.edu/files/fits/files/fits-0.8.4.zip 
-fits_ver: 0.8.4
 
```

Comment:
The fits_url variable was there for projects that are still using fits 0.6.x, since the hosting for fits changed between those two versions.